app/src/androidTest/java/ch/epfl/sdp/appart/AdCreationUITest.java

```diff
@@ -86,6 +86,8 @@ public void init() {
         login.loginWithEmail("lorenzo@epfl.ch", "2222");
     }
 
+    /** Error on Cirrus...
+     *
     @Test
     public void viewsAreDisplayedTest() {
```

Comment:
I think this test can entirely be removed as it doesn't seem to change the coverage.

Reply:
yess I've removed it!